sdk/storage/tests.yml

```diff
@@ -4,6 +4,7 @@ stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: storage
+      SupportedClouds: 'Public,Canary,UsGov,China'
```

Comment:
1. Can we please not add this to nightly runs? This will bring more flaky-ness. Instead I suggest to make it part of weekly pipe https://dev.azure.com/azure-sdk/internal/_build?definitionId=2714 where we already have few clouds listed. Regressions that are sovereign cloud specific are rare.
2. Storage tests are running on dedicated agent pool in canada central region where we provision test accounts (see line `Pool=(.*)-general/$1-storage` below). We heavily rely on that co-location to run tests with larger payloads. A mechanism to disable these tests in sovereign clouds should be developed first or we should get agents that are co-located there.

Reply:
This change has been rolled back